app/src/main/java/com/sdpteam/connectout/event/nearbyEvents/filter/EventParticipantIdFilter.java

```diff
@@ -19,6 +19,6 @@ public boolean test(Event event) {
         if (userId == null) {
             return false;
         }
-        return userId.equals(event.getOrganizer()) || event.getParticipants().contains(userId);
+        return userId.equals(event.getOrganizer()) || event.hasJoined(userId);
```

Comment:
Interesting. So are you saying that a user that has "joined" should be considered in the filtering? please enlighten me on why this should be the case

Reply:
It does the same, it's just refactoring